tests/core/go_binary/BUILD.bazel

```diff
@@ -67,3 +67,30 @@ go_library(
         "DepSelf": "DepSelf",
     },
 )
+
+go_binary(
+    name = "hello_pie_bin",
+    srcs = ["hello.go"],
+    cgo = True,
+    linkmode = "pie",
+)
+
+go_binary(
+    name = "hello_nopie_bin",
+    srcs = ["hello.go"],
+    cgo = True,
+)
+
+go_test(
+    name = "pie_test",
+    srcs = [
+        "pie_darwin_test.go",
+        "pie_linux_test.go",
+    ],
+    data = [
```

Comment:
It looks like there is an error in Windows CI because the linker complains when `-buildmode=pie` is passed to it. This is not a great error message, but it's consistent with what we do for other modes, so no need to change it in this PR.

I'd suggest adding `tags = ["manual"]` to both `go_binary` rules above, then changing this `data` attribute to not pull anything in on Windows.

Something like this should work:

```
data = select({
    "@io_bazel_rules_go//go/platform:darwin": [
        ":hello_nopie_bin",
        ":hello_pie_bin",
    ],
    "@io_bazel_rules_go//go/platform:linux": [
        ":hello_nopie_bin",
        ":hello_pie_bin",
    ],
    "//conditions:default": [],
}),
```

Reply:
Yep, That works. Thanks for the suggestion.